client/src/pages/SearchStart.js

```diff
@@ -1,14 +1,14 @@
-import { FormInput } from "../components/Form";
 import { List } from "../components/ListItem";
-import IconSearch from "../assets/icon-search-primary.svg";
 import IconArrowForward from "../assets/icon-arrow-forward-primary.svg";
 import Header from "../components/Header";
 import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import { getAllResults } from "../api/search";
+import { InputField } from "../components/Input";
 
 export default function Search() {
   const [results, setResults] = useState([]);
+  const [searchFilter, setSearchFilter] = useState([]);
```

Comment:
then maybe because of this line?
```suggestion
  const [searchFilter, setSearchFilter] = useState("");
```

Reply:
Now it works!!! 🎉  🥳   Great job! Thanks a lot!!!